sirepo/auth/__init__.py

```diff
@@ -665,6 +665,14 @@ def _parse_display_name(value):
     return res
 
 
+def _plan(roles):
+    if 'enterprise' in roles:
+        return 'enterprise'
```

Comment:
Are we returning the role name or is that decoupled from the constants list and this is a different name. In any event, it should be validated against the schema at some point. Could be at init.

Reply:
For premium and enterprise they are the same as the role name. There isn't a basic role but there is a basic paymentPlan so I think it is decoupled. Added the checking in init

https://github.com/radiasoft/sirepo/pull/2711/commits/110ea9c3b63ca85561bac84b1dd582a383edab18